query/sns/sns_topic_notification_delivery_status_enabled.sql

```diff
@@ -0,0 +1,24 @@
+select
+  -- Required Columns
+  topic_arn as resource,
+  case
+    when application_failure_feedback_role_arn is null 
+      and firehose_failure_feedback_role_arn is null 
+      and http_failure_feedback_role_arn is null 
+      and lambda_failure_feedback_role_arn is null 
+      and sqs_failure_feedback_role_arn is null then 'alarm'
+    else 'ok'
+  end as status,
+  case
+    when application_failure_feedback_role_arn is null 
+      and firehose_failure_feedback_role_arn is null 
+      and http_failure_feedback_role_arn is null 
+      and lambda_failure_feedback_role_arn is null 
+      and sqs_failure_feedback_role_arn is null then title || ' has delivery status logging for notification messages enabled.'
+    else title || ' has delivery status logging for notification messages disabled.'
```

Comment:
I think the status and reason columns have been reversed. For instance, the `alarm` status is going to have the reason that `title || ' has delivery status logging for notification messages enabled.`